.github/workflows/ci.yml

```
@@ -42,6 +42,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: EmbarkStudios/cargo-deny-action@v2
 
+  cargo-machete:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bnjbvr/cargo-machete@v0.6.2
```

Cache to make this fast.
```suggestion
      - uses: Swatinem/rust-cache@v2
      - uses: bnjbvr/cargo-machete@v0.6.2
```